.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this be in the .gitignore? I've added it in #712 so if we merge that first, it's already there.

Reply:
You're right, I'll merge #713 first